streams-tests/jvm/src/test/scala/zio/stream/StreamSpec.scala

```diff
@@ -1160,27 +1161,14 @@ class ZStreamSpec(implicit ee: org.specs2.concurrent.ExecutionEnv)
     val s2 = Stream(1, 2)
 
     val merge = s1.mergeWith(s2)(_.toString, _.toString)
-    val list: List[String] = slurp(merge).toEither.fold(
+    val list: List[String] = unsafeRunSync(merge.runCollect).toEither.fold(
       _ => List.empty,
       identity
     )
 
     list must containTheSameElementsAs(List("1", "2", "1", "2"))
   }
 
-  private def mergeWithShortCircuit = {
```

Comment:
Why is this test being dropped completely? Should we put a TODO or something?

Reply:
Initially I thought it's not an important test when we switch to `process`, but I guess more coverage doesn't hurt.